src/app/external-cluster-wizard/steps/summary/style.scss

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'src/assets/css/variables';
+@use 'src/assets/css/mixins';
+
+$steps: 3;
+
+@mixin counter-x {
+  @for $i from 1 through $steps {
+    .counter-#{$i} {
+      &::before {
+        @include mixins.size(22px);
+
+        align-items: center;
+        border-radius: 50%;
+        border-style: solid;
+        border-width: 1px;
+        content: '#{$i}';
```

Comment:
This is something new what it does?
`content: #{$i}`

Reply:
It adds the number of the current steps

<img width="310" alt="image" src="https://user-images.githubusercontent.com/56511988/188610784-21c073f3-04b6-4afc-871a-8f725ba7be34.png">

Like here it's adding (1) and (2).